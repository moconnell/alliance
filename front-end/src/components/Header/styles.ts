import styled from "styled-components";

export const Container = styled.div`
  margin-top: 42px;
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: end;
  padding: 0px 43px;
`;
