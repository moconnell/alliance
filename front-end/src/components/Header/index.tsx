import Web3Button from "../Web3Button";
import { Container } from "./styles";

const Header = () => (
  <Container>
    <Web3Button />
  </Container>
);

export default Header;
