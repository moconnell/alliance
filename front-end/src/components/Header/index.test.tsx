import { render } from "@testing-library/react";
import Header from "./index";

jest.mock("../Web3Button", () => () => "((Web3Button))");

describe("<Header />", () => {
  it("should render correctly", async () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
