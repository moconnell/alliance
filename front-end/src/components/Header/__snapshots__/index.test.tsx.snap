// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ lkRbBT"
  >
    ((Web3Button))
  </div>
</DocumentFragment>
`;
